frontend/src/Login/ForgottenPassword.jsx

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable no-useless-escape */
+import React, { useState } from 'react';
+import {
+  View, StyleSheet, Text,
+  ImageBackground, Alert,
+} from 'react-native';
+import { useDispatch } from 'react-redux';
+import { useNavigation } from 'react-navigation-hooks';
+
+import { InputField, SubmitButton } from '../common/components';
+import { forgotPassword } from './actionCreator';
+import background from '../../assets/login/background.jpg';
+import Colors from '../common/colors';
+
+const styles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+    justifyContent: 'center',
+    padding: 0,
+  },
+  header: {
+    width: 300,
+    marginVertical: 12,
+    fontSize: 24,
+    fontWeight: 'bold',
+    color: Colors.whiteColor,
+  },
+  background: {
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  button: {
+    flexDirection: 'row',
+    marginTop: 20,
+    justifyContent: 'space-between',
+  },
+});
+
+function showAlert(text) {
+  Alert.alert('Warning', text);
+}
+
+function ForgottenPassword() {
+  const navigation = useNavigation();
+  const [input, setInput] = useState('');
+  const dispatch = useDispatch();
+  const USERNAME_BLACKLIST = ['admin', 'root'];
+
+  function validation(email) {
+    const emailChecker = new RegExp([
+      '^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)',
+      '|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])',
+      '|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$',
+    ].join(''));
+    return emailChecker.test(String(email).toLowerCase());
+  }
+
+  function handleSubmit() {
+    if (input === '') {
+      return showAlert('The input field is required.');
+    }
+    if (input.length <= 3) {
+      return showAlert('Please enter a valid username.');
+    }
+    if (!validation(input) && USERNAME_BLACKLIST.indexOf(input.toLowerCase()) !== -1) {
```

Comment:
We can use es6 ways.
```suggestion
    if (!validation(input) && USERNAME_BLACKLIST.includes(input.toLowerCase())) {
```